knpu/events.md

```diff
@@ -71,6 +77,13 @@ we can actually dream up whatever name we want. Let's use:
 
 [[[ code('19be25e81c') ]]]
 
+***TIP
+Starting from Symfony 4.4, you should use instead:
+```php
+$this->eventDispatcher->dispatch($event, 'knpu_lorem_ipsum.filter_api');
```

Comment:
I'm going to change this to go even further:

```
***TIP
Starting in Symfony 4.4, you only need to pass the `$event` argument:

```php
$this->eventDispatcher->dispatch($event);
```

Then, instead of `knpu_lorem_ipsum.filter_api`, the event name becomes the event class:
in our  case `FilterApiResponseEvent::class`.
```